### PR TITLE
Add PoolPositionModal component

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

**Comment by danieledamiani** [comment]
in review

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 1]
```diff
@@ -98,6 +98,10 @@
     "totalYieldEarned": "",
     "projectedTotalYield": ""
   },
+  "PoolPositionModal": {
+    "title": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:102]: `Dettagli Posizione`

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 2]
```diff
@@ -98,6 +98,10 @@
...
+    "withdrawAction": ""
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:103]: `Ritira`

File: packages/tempus-client_v3/src/components/shared/ActionButton/ActionButton.tsx [hunk 3]
```diff
@@ -10,9 +10,9 @@ type ActionButtonSize = 'small' | 'large';
 type ActionButtonVariant = 'primary' | 'secondary' | 'tertiary';
 export type ActionButtonState = 'default' | 'disabled' | 'loading' | 'success';
 export type ActionButtonLabels = {
-  default: string;
-  loading: string;
-  success: string;
+  default?: string;
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/components/shared/ActionButton/ActionButton.tsx:13]: if no property is defined the label will be empty, does this cause issues with rendering?

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 4]
```diff
@@ -98,6 +98,10 @@
     "totalYieldEarned": "",
     "projectedTotalYield": ""
   },
+  "PoolPositionModal": {
+    "title": "",
+    "withdrawAction": ""
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:103]: `Retirar`

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 5]
```diff
@@ -98,6 +98,10 @@
     "totalYieldEarned": "",
     "projectedTotalYield": ""
   },
+  "PoolPositionModal": {
+    "title": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:102]: `Detalles Posición`

**Review by danieledamiani — APPROVED** [review]

File: packages/tempus-client_v3/src/components/shared/ActionButton/ActionButton.tsx (hunk 3 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/shared/ActionButton/ActionButton.tsx:13]: Yeah, I changed default label to be mandatory again.

**Comment by lazarmitic** [comment]
All comments resolved.

**Pull request merged**